frontend/src/main/web/src/components/LanguageTeamTeaser.jsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react' // eslint-disable-line
+import Link from '../components/Link'
+import View from '../components/View'
+import Icon from '../components/Icon'
+// import { flattenClasses } from '../utils'
+
+const viewTheme = {
+  base: {
+    ai: 'Ai(c)',
+    fld: '',
+    m: 'Mb(rh)'
+  }
+}
+
+const LanguageTeamTeaser = ({
+  children,
+  name,
+  details,
+  ...props
+}) => {
+  const useHref = true
+  const link = useHref
+    ? window.config.baseUrl + '/language/view/' + details.id
+    : 'language/' + details.id
```

Comment:
Unreachable code since `useHref` is hard-coded to `true`. Replace ternary with simple expression, or fix the hard-coded value on `useHref` if is it a mistake.